src/wp-admin/options.php

```diff
@@ -179,11 +190,8 @@
 
 	$allowed_options['general'][] = 'users_can_register';
 	$allowed_options['general'][] = 'default_role';
-
-	$allowed_options['writing']   = array_merge( $allowed_options['writing'], $mail_options );
 	$allowed_options['writing'][] = 'ping_sites';
-
-	$allowed_options['media'][] = 'uploads_use_yearmonth_folders';
+	$allowed_options['media'][]   = 'uploads_use_yearmonth_folders';
```

Comment:
I don't think these changes are in the scope of this update and would suggest leaving them out for now. The groupings here look intentional.

Reply:
done

Reply:
done